package.json

```diff
@@ -574,7 +582,7 @@
     "dependencies": {
         "@azure/arm-authorization": "^9.0.0",
         "@azure/arm-containerregistry": "^10.1.0",
-        "@azure/arm-containerservice": "^21.1.0-beta.1",
+        "@azure/arm-containerservice": "21.2.0-beta.1",
```

Comment:
💡 Side note: Final eye balling: `^` carrot missing here,

`^version` “Compatible with version”, will automatically update you to all future minor/patch versions that are backwards-compatible, without incrementing the major version. For example:  `^1.2.3 will use releases from 1.2.3 to < 2.0.0`.